frontend/nuxt.config.ts

```diff
@@ -26,6 +22,19 @@ export default defineNuxtConfig({
         de: "/räume/erstellen",
         en: "/rooms/create",
       },
+      play: {
+        de: "/play",
+        en: "/play",
+      },
+      "play-[roomcode]": {
+        de: "/play/[roomcode]",
+        en: "/play/[roomcode]",
+      },
+      add_team_members: {
+        de: `/play/mitglieder`,
```

Comment:
da reicht auch ein normaler string
```suggestion
        de: "/play/mitglieder",
```